src/bin.djs

```diff
@@ -3,7 +3,7 @@ so ./index as doge3
 
 very file is 0
 rly process giv argv giv length bigger 2
-    file is process giv argv levl 2
+  file is process giv argv levl 2
```

Comment:
does this change have any significance? 4 spaces is the convention per the 2.x spec document

Reply:
Ah, no, i just put a `tab` in there on accident when mucking with it and then converted tabs to 2 space, lemme rever that.